examples/flutter_gallery/lib/gallery/drawer.dart

```diff
@@ -2,6 +2,9 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:math' as math;
+import 'dart:ui' as ui show window;
```

Comment:
Most of our example code uses MediaQuery.of(context).padding instead of window.ui.padding. We should be consistent.